evap/staff/views.py

```diff
@@ -144,159 +149,194 @@ def __init__(self):
     return render(request, "staff_semester_view.html", template_data)
 
 
+class EvaluationOperation:
+    email_template_name = None
+    confirmation_message = None
+
+    @staticmethod
+    def applicable_to(evaluation):
+        raise NotImplementedError
+
+    @staticmethod
+    def warning_for_inapplicables(amount):
+        return ungettext("{} evaluation can not be updated. It was removed from the selection.",
+            "{} evaluations can not be updated. They were removed from the selection.", amount).format(amount)
```

Comment:
`raise NotImplementedError`?

Reply:
Fits the rest of the class more - I agree.